.github/workflows/pull-request-test.yml

```
@@ -0,0 +1,188 @@
+            git status # Debug: Show what's staged
+            git commit -m "Update visual test diffs for PR #${PR_NUMBER} (Run #${{ github.run_id }})"
+            git push -f origin ${BRANCH_NAME}
+            echo "DIFFS_BRANCH=${BRANCH_NAME}" >> $GITHUB_ENV
+            echo "✅ Successfully pushed diff images to branch"
+          else
+            echo "No diff images found"
+          fi
+
+      - name: Comment on PR with failure details
+        if: always()
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)